server/chat-plugins/trivia.ts

```diff
@@ -344,17 +348,12 @@ class Trivia extends Rooms.RoomGame {
 
 	/**
 	 * How long the players should have to answer a question.
-	 * @return {number}
 	 */
 	getRoundLength() {
 		return 12 * 1000 + 500;
 	}
 
-	/**
-	 * @param {User} user
-	 * @return {string | undefined}
-	 */
-	addPlayer(user) {
+	addTriviaPlayer(user: User) {
```

Comment:
Why was this renamed?

Reply:
`addPlayer` is already a function in `Room.RoomGame`, and this function was attempting to mutate the return of the base function. This modifies it to allow for the proper return statements required for error checking when a user tries to enter a Trivia game and still adds the player to the game; renaming was largely unproblematic because It already called `super.addPlayer` if the inputted user passed all error checking.

Reply:
I suppose this isn't a bad solution, but I wonder if exceptions would be better.